packages/cli/scripts/generate-test-tables.ts

```diff
@@ -16,10 +15,10 @@ const config = defineStore({
     Enum2: ["E1"],
   },
   userTypes: {
-    TestTypeAddress: { filePath: "./contracts/src/types.sol", type: "address" },
-    TestTypeInt64: { filePath: "./contracts/src/types.sol", type: "int64" },
-    TestTypeBool: { filePath: "./contracts/src/types.sol", type: "bool" },
-    TestTypeUint128: { filePath: "./contracts/src/types.sol", type: "uint128" },
+    TestTypeAddress: { filePath: "../contracts/src/types.sol", type: "address" },
+    TestTypeInt64: { filePath: "../contracts/src/types.sol", type: "int64" },
+    "TestTypeLibrary.TestTypeBool": { filePath: "../contracts/src/types.sol", type: "bool" },
+    "TestTypeLibrary.TestTypeUint128": { filePath: "../contracts/src/types.sol", type: "uint128" },
```

Comment:
previously, codegen would parse the file at `filePath` to determine where in the file was the specified user type and then propagate the fully qualified name everywhere

this felt too "automagic" to me and added a lot of complexity to codegen that I was able to remove by not automatically inferring the code path for the user type (with the nice side effect that we no longer need remappings for tablegen)

we can still support this (imo rare) case by specifying the fully qualified type name like above

(maybe in the future could improve this with a similar "label" approach where the label is used in config references like schemas, and allow specifying a fully qualified code path override of `internalType` that gets used in codegen)